scripts/exregional_run_analysis.sh

```diff
@@ -530,27 +602,80 @@ for file in $(awk '{if($1!~"!"){print $1}}' ./satinfo | sort | uniq) ;do
    ln -s ${CRTMFIX}/${file}.TauCoeff.bin ./
 done
 
-## satellite bias correction
-#if [ ${FULLCYC} -eq 1 ]; then
-#   latest_bias=${DATAHOME_PBK}/satbias/satbias_out_latest
-#   latest_bias_pc=${DATAHOME_PBK}/satbias/satbias_pc.out_latest
-#   latest_radstat=${DATAHOME_PBK}/satbias/radstat.rap_latest
-#fi
-
-# cp $latest_bias ./satbias_in
-# cp $latest_bias_pc ./satbias_pc
-# cp $latest_radstat ./radstat.rap
-# listdiag=`tar xvf radstat.rap | cut -d' ' -f2 | grep _ges`
-# for type in $listdiag; do
-#       diag_file=`echo $type | cut -d',' -f1`
-#       fname=`echo $diag_file | cut -d'.' -f1`
-#       date=`echo $diag_file | cut -d'.' -f2`
-#       gunzip $diag_file
-#       fnameanl=$(echo $fname|sed 's/_ges//g')
-#       mv $fname.$date $fnameanl
-# done
+#-----------------------------------------------------------------------
 #
-#mv radstat.rap  radstat.rap.for_this_cycle
+# cycling radiance bias corretion files
+#
+#-----------------------------------------------------------------------
+if [ ${DO_RADDA} == "TRUE" ]; then
+  if [ ${cycle_type} == "spinup" ]; then
+     if [ ${HH} -eq '03' ] || [ ${HH} -eq '15' ]; then
+     spinup_or_prod_rrfs=prod
+     else
+     spinup_or_prod_rrfs=spinup
+     fi
+  elif [ ${cycle_type} == "prod" ]; then
+     if [ ${HH} -eq '09' ] || [ ${HH} -eq '21' ]; then
+      spinup_or_prod_rrfs=spinup
+     else
+      spinup_or_prod_rrfs=prod
+     fi
+  fi
+
+  satcounter=1
+  while [[ $satcounter -lt 240 ]]; do
+    SAT_TIME=`date +"%Y%m%d%H" -d "${START_DATE}  ${satcounter} hours ago"`
+    echo $SAT_TIME
+    if [ -r ${SATBIAS_DIR}/rrfs.${spinup_or_prod_rrfs}.${SAT_TIME}_satbias ]; then
+      echo " using satellite bias files from ${SAT_TIME}"
+
+      cp_vrfy ${SATBIAS_DIR}/rrfs.${spinup_or_prod_rrfs}.${SAT_TIME}_satbias ./satbias_in
+      cp_vrfy ${SATBIAS_DIR}/rrfs.${spinup_or_prod_rrfs}.${SAT_TIME}_satbias_pc ./satbias_pc
+      cp_vrfy ${SATBIAS_DIR}/rrfs.${spinup_or_prod_rrfs}.${SAT_TIME}_radstat ./radstat.rrfs
+
+      break
+    fi
+    satcounter=` expr $satcounter + 1 `
+  done
+
+  #echo $satcounter
+
+  ## if satbias files (go back to previous 10 dyas) are not available from ${SATIBAS_DIR}, use satbias files from the ${FIX_GSI} 
+  if [ $satcounter -eq 240 ]; then
+
+     if [ ! -d "${SATBIAS_DIR}" ]; then
+       echo "makding satbias_dir directory for cycling bias correction files" 
+       mkdir -p ${SATBIAS_DIR}
+     fi
```

Comment:
SATBIAS_DIR should have subdirectory and it should be created in J-Job

Reply:
I put this in the J-job script.